lib/mogrify.ex

```diff
@@ -259,6 +261,13 @@ defmodule Mogrify do
     %{image | operations: image.operations ++ [quality: params]}
   end
 
+  @doc """
+  Adds support for limit in the format of array, eg. ["memory", "30Mib"]
+  """
+  def limit(image, params) do
```

Comment:
This should probably be split to 2 alternative functions, one `def limit(image, type, value)` and one `def limit(image, params)` where params is a binary like `"memory 30MiB"` i.e. `"#{type} #{value}"`

This is for consistency with the other functions since the other ones take binary `params`, not arrays.

Then normalize_arguments can be:

    defp normalize_arguments({:limit, params}), do: ~w(-limit #{params})

since `~w` splits the string into an array by whitespace.